ta/mk/ta_dev_kit.mk

```diff
@@ -8,6 +8,7 @@ all:
 include $(ta-dev-kit-dir)/mk/conf.mk
 
 binary := $(BINARY)
+libname := $(LIBNAME)
```

Comment:
We should perhaps have some user friendly error checking in case both `BINARY` and `LIBNAME` are holding values.